src/App.tsx

```diff
@@ -1,34 +1,91 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { actions as actionTodos } from './features/todos';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { SortType } from './types/SortTypes';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector((state) => state.todos);
+  const filterState = useAppSelector((state) => state.filter);
+  const selectTodo = useAppSelector((state) => state.currentTodo);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        const loadedTodos = await getTodos();
+
+        dispatch(actionTodos.SetAdd(loadedTodos));
+        setIsLoading(false);
+      } catch (error) {
+        if (error instanceof Error) {
```

Comment:
what if it's not?
than you should show some error enyway

Reply:
Still not fixed